src/config/params.rs

```diff
@@ -91,6 +91,8 @@ impl<'a> From<&'a Config> for BgParams {
             height: config.height,
             radius: config.corner_radius.clone(),
             color: config.bg_color.unwrap_or(DEFAULT_BG_COLOR),
+            border_color: config.bg_border_color.unwrap_or(DEFAULT_BG_BORDER_COLOR),
+            border_width: config.bg_border_width.unwrap_or(2.0),
```

Comment:
It make sense to leave the default behavior as-is, i.e do not draw any border

Reply:
Yeah, agreed, changed it to `0.0`